velox/core/PlanNode.cpp

```diff
@@ -327,6 +546,47 @@ void UnnestNode::addDetails(std::stringstream& stream) const {
   addFields(stream, unnestVariables_);
 }
 
+folly::dynamic UnnestNode::serialize() const {
+  auto obj = PlanNode::serialize();
+  obj["replicateVariables"] = ISerializable::serialize(replicateVariables_);
+  obj["unnestVariables"] = ISerializable::serialize(unnestVariables_);
+
+  obj["unnestNames"] = folly::dynamic::array();
+
+  for (auto i = replicateVariables_.size();
```

Comment:
maybe a tiny one-liner comment on why we need this?

Reply:
Added.